modules/internal/Atomics.chpl

```diff
@@ -700,35 +700,35 @@ module Atomics {
   // We need to explicitly define these for all types because the atomic
   // types are records and unless explicitly defined, it will resolve
   // to the normal record version of the function.  Sigh.
-  inline proc =(ref a:AtomicBool, const ref b:AtomicBool) {
+  inline operator =(ref a:AtomicBool, const ref b:AtomicBool) {
```

Comment:
I would imagine that we'd want these atomic operators to be operator methods (associated with the atomic type).

Reply:
This mostly was fine except one test failed for AtomicT:

```
[Elapsed compilation time for "library/standard/Types/copyable" - 2.882 seconds]
[Executing diff copyable.good copyable.comp.out.tmp]
0a1,3
> copyable.chpl:50: In function 'checkNormal':
> copyable.chpl:51: error: isConstAssignable AtomicT(int(64)) did not match
>   copyable.chpl:117: called as checkNormal(type t = AtomicT(int(64)), e: Atomi\
cT(int(64)))
[Error matching compiler output for library/standard/Types/copyable]
```

As a result I'm going to track this as something to come back to later and investigate.